typed_python/compiler/type_wrappers/runtime_functions.py

```diff
@@ -333,6 +333,12 @@ def unaryPyobjCallTarget(name, retType=Void.pointer()):
     UInt64, Void.pointer(), Void.Pointer(), Int64
 )
 
+class_cmp = externalCallTarget(
```

Comment:
if __cmp__ is not defined, class comparison throws an exception for ordered comparison, and for EQ/NEQ, it just uses the class pointer. We can do the same thing in compiled code.

Reply:
This has been done.